src/Roots/Acorn/Assets/Bundle.php

```diff
@@ -125,7 +125,7 @@ protected function getUrl($path)
             return $path;
         }
 
-        return "{$this->uri}/${path}";
```

Comment:
Tell me you type template literals without telling me you type template literals.